src/components/button/button.component.tsx

```diff
@@ -9,6 +9,7 @@ import { TooltipProvider } from "../../__internal__/tooltip-provider";
 import Logger from "../../__internal__/utils/logger";
 import { TooltipPositions } from "../tooltip/tooltip.config";
 import { ButtonBarContext } from "../button-bar/button-bar.component";
+import splitButtonContext from "../split-button/split-button.context";
```

Comment:
**nitpick**: `splitButtonContext` -> `SplitButtonContext`

Reply:
Oh yeah, I was thinking uppercase was only for components but not sure why I didn't think to check other contexts that we define and use in Carbon 🤦